StortSpelProjekt/Game/src/main.cpp

```diff
@@ -6,9 +6,14 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
 {
     _CrtSetDbgFlag(_CRTDBG_ALLOC_MEM_DF | _CRTDBG_LEAK_CHECK_DF);
 
+	// Window values (will be influenced by the option class)
+	bool fullscreen = false;
+	int screenWidth = 1920;
+	int screenHeight = 1080;
```

Comment:
When you change this, you modify both the window size + the resolution of the renderTargets. Is this what you meant to do? This means that we will not be able to test the game in "high-res" without going fullscreen.

Reply:
Yes, that is what I meant to do. However, you do have a valid point and I will add a resolutionWidth and a resolutionHeight as well